### DISCUSSION TITLE
New device-types request

### DISCUSSION THREAD

**Discussion by paulng30** [netbox-community/devicetype-library#1550] [Q&A] [answered]
Hi,

I'm looking for Cisco UCS X9508 chassis YAML file for adding to device types.  How may I request for Cisco UCS X9508 chassis YAML file to be added to Netbox devicetype-library?  

Thanks,
Paul

**Comment by DanSheps** [comment, marked as answer]
You can create it.

This is a community sourced repository so files won't be created unless someone within the community creates it.

**Comment by JustAnotherAdmin23** [comment]
Got you boo. <3

https://github.com/netbox-community/devicetype-library/blob/master/device-types/Cisco/UCS-X9508.yaml

**Comment by satochir** [comment]
Hi, Does anyone know if there is a category for xfusion devices i couldn't find it in the list.

**Reply by DanSheps** [reply]
If not, you are free to create it
